stripper/ze_surf_sahok_p3.cfg

```diff
@@ -3,12 +3,12 @@ modify:
 {
 	match:
 	{
-		"classname" "math_counter"
-		"targetname" "Boss_Health"
+		"classname" "func_physbox"
+		"targetname" "Boss_Physbox"
 	}
-	insert:
+	replace:
 	{
-		"min" "0"
+		"health" "999999999"
```

Comment:
Boss_Physbox is not used to track HP at all, nor is it in the bhud config. This won't fix anything.